content/6_Advanced/Multiplicative.mdx

```diff
@@ -1,24 +1,227 @@
+
+Common multiplicative functions are
+
+- Divisor function: $\sigma_k(n) = \sum_{d|n} d^k$, representing the sum of the
+  $k$th powers of divisors of $n$. Note that $\sigma_k(n)$ and $σ^k(n)$ are
+  different.
+- Divisor count function: $\tau(n) = \sigma_0(n) = \sum_{d|n} 1$, representing
+  the count of divisors of $n$, also denoted as $d(n)$.
+- Divisor sum function: $\sigma(n) = \sigma_1(n) = \sum_{d|n} d$, representing
+  the sum of divisors of $n$.
+- Euler's totient function: $\varphi(n) = \sum_{i=1}^n [(n,i)=1] \cdot 1$,
+  representing the count of positive integers less than or equal to $n$ and
+  coprime to $n$. Additionally,
+  $\sum_{i=1}^n [(n,i)=1] \cdot i = $ $$\frac{n\varphi(n) +
+  [n=1]}{2}$$, $\varphi(n)$ is even.
+- Möbius function: $\mu(n)$, serving as the multiplicative inverse of the
+  identity function in Dirichlet convolution, $\mu(1) = 1$, for a square-free
+  number $n = \prod_{i=1}^t p_i$, $\mu(n) = (-1)^t$, and for a number with
+  square factors, $\mu(n) = 0$.
+- Unit function: $e(n) = [n=1]$, serving as the identity element in Dirichlet
+  convolution, completely multiplicative.
+- Constant function: $I(n) = 1$, completely multiplicative.
+- Identity function: $id(n) = n$, completely multiplicative.
+- Power function: $id^k(n) = n^k$, completely multiplicative.
+
+The two classic formulas regarding the Möbius function and the Euler function
+are:
+
+- $$[n=1] = \sum_{d|n} \mu(d)$$, Interpreting $\mu(d)$ as the coefficients of
+  the inclusion-exclusion principle proves it.
+- $$ n = \sum_{d|n} \varphi(d) $$. To prove it, we can count the number of occurrences of $$\frac{1}{n}(1 \leq i \leq n)$$ in its simplest fraction form.
+
+
+### Dirichlet Convolution
+
+The Dirichlet convolution of number-theoretic functions $f$ and $g$ is defined
+as $(f*g)(n) = \sum_{d|n} f(d) \cdot g(\frac{n}{d})$. Dirichlet convolution
+satisfies commutativity, associativity, and distributivity with respect to
+addition. There exists an identity function $e(n) = [n=1]$ such that
+$f*e = f = e*f$。If $f$ and $g$ are multiplicative functions, then $f*g$ is also
+multiplicative.
+
+A common technique with Dirichlet convolution involves dealing with the
+convolution of a multiplicative function $f$ and the identity function $I$. For
+example, if $n = \prod_{i=1}^{t} p_i^{k_i}$ and $g(n) = \sum_{d|n} f(d)$，then
+we have
+
+$$
+g(n) = \prod_{i=1}^{t} \sum_{j=0}^{k_i} f(p_i^j)
+$$
+
+### Möbius Inversion
+
+Möbius inversion is also a discussion regarding $g(n) = \sum_{d|n} f(d)$ , but
+it does not require $f$ to be multiplicative and is applicable in cases where
+$g(n)$ is known and $f(n)$ is to be determined.
+
+Since $I \cdot \mu = e$，
+$g \cdot \mu = f \cdot I \cdot \mu = f \cdot e = f$，and
+$f(n) = \sum_{d|n} g(d) \cdot \mu\left(\frac{n}{d}\right)$. Similarly,
+$g(n) = \sum_{n|d} f(d) \rightarrow f(n) = \sum_{n|d} g(d) \cdot \mu\left(\frac{d}{n}\right)$，
+
+Binomial inversion is also a similar technique. An example illustrates the
```

Comment:
do you have a source for this being called binomial inversion?

binomial inversion seems like something different ...

https://math.stackexchange.com/questions/55659/combinatorial-interpretation-of-binomial-inversion#:~:text=It%20is%20known%20that%20if,binomial%20inversion%2C%20for%20obvious%20reasons.

Reply:
Can lines 98 - 102 be removed? I don't see how this information is useful.